src/test/java/uk/gov/service/notify/ClientIntegrationTestIT.java

```diff
@@ -279,5 +279,59 @@ private void assertNotificationWhenSms(Notification notification) {
         assertFalse(notification.getPostcode().isPresent());
     }
 
+    @Test
+    public void testEmailNotificationWithValidEmailReplyToIdIT() throws NotificationClientException, InterruptedException {
+        NotificationClient client = getClient();
+        SendEmailResponse emailResponse = sendEmailAndAssertResponse(client);
+
+        HashMap<String, String> personalisation = new HashMap<>();
+        String uniqueName = UUID.randomUUID().toString();
+        personalisation.put("name", uniqueName);
+
+        SendEmailResponse response = client.sendEmail(
+                System.getenv("EMAIL_TEMPLATE_ID"),
+                System.getenv("FUNCTIONAL_TEST_EMAIL"),
+                personalisation,
+                uniqueName,
+                System.getenv("EMAIL_REPLY_TO_ID"));
+
+        assertNotificationEmailResponse(response, uniqueName);
+
+        Notification notification = client.getNotificationById(emailResponse.getNotificationId().toString());
+        assertNotification(notification);
```

Comment:
- Do you need to assert that the notification is sent with the reply_to_address other than the default?
- You may also want to add the new field in assertNotification()